File: src/AoC16/s16e01-csharp/Part1.cs
using System;

namespace AoC16S16E01
{
    class Part1
    {
        static void Main()
        {
            string input = Console.In.ReadToEnd().Trim();
            int result = Solution.SolvePart1(input);
            Console.WriteLine(result);
        }
    }
}

File: src/AoC16/s16e01-swift/Part1.swift
import Foundation

@main
struct Part1 {
    static func main() {
        var input = ""
        while let line = readLine() {
            input += line + "\n"
        }
        input = input.trimmingCharacters(in: .whitespacesAndNewlines)
        let result = solvePart1(input)
        print(result)
    }
}
